tests/e2e/tests/test_clustering.py

```diff
@@ -48,3 +48,23 @@ def test_worker_nodes(instances: List[harness.Instance]):
     join_cluster(joining_node, token)
 
     util.wait_until_k8s_ready(cluster_node, instances)
+
+
+@pytest.mark.node_count(4)
+def test_control_plane_replacement(instances: List[harness.Instance]):
+    cp1, cp2, cp3, cp_r = instances
```

Comment:
What is a cp_r ? cp1, cp2, cp3 are obvious enough but I'm not sure cp_r is clear.

Reply:
CPR is "Cardiopulmonary Resuscitation". 

but in this case it's "Control-Plane Replacement" -- the instance i started to replace the one i'm going to remove